### PR TITLE
add tag filter fga-eps-mds/2021.1-Oraculo#223

### PR DESCRIPTION
**Pull request by joaoCeilandia**

### CODE REVIEW

File: src/Controller/RecordController.js [hunk 1]
```diff
@@ -149,10 +149,16 @@ async function getRecordsByPage(req, res) {
...
 
-    const { exact = {}, history, ..._where } = where || {};
+    const tagFields = [
+      'name',
+      'color'
+    ]
+
+    const { history, tag, ..._where } = where || {};
 
     let filters = {};
```
Comment (lucas-engen) [inline src/Controller/RecordController.js:159]: Alterar `let filters = {}` para:

```javascript
const filters = {}
```

File: src/Controller/RecordController.js [hunk 2]
```diff
@@ -168,11 +174,18 @@ async function getRecordsByPage(req, res) {
       });
     }
 
-    filters = { ...exact, ...filters };
+    if(tag) {
+      tagFields.forEach((item) => {
+        tagFilters.push({[item]: {
+          [Op.iLike]: "%" + tag + "%"
```
Comment (lucas-engen) [inline src/Controller/RecordController.js:180]: Alterar `[Op.iLike]: "%" + tag + "%"` para

```javascript
[Op.iLike]: `%${tag}%`
```

**Review by lucas-engen — DISMISSED** [review]

**Review by ericoBandeira — APPROVED** [review]

**Review by VictorRodriguesS0 — APPROVED** [review]

**Review by jpaulohe4rt — APPROVED** [review]
🆗

**Review by jpaulohe4rt — APPROVED** [review]
🆗

**Pull request merged**